### PR TITLE
PLAT-126340: [Agate] Added UI and unit tests for Keypad

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for Keypad
 - Added unit tests for Keypad

### Links
[//]: # (Related issues, references)
PLAT-126340

### Comments

### CODE REVIEW

File: tests/ui/specs/Keypad/KeypadPage.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (stanca-pop-lgp) [inline tests/ui/specs/Keypad/KeypadPage.js:1]: You have 2 empty lines. Was that intended?

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]

**Review by jeonghee27 — COMMENTED** [review]
LGTM.
It's only verify the focus in ui-test, but there is a unit-test and there is no `Keypad` TC currently.
So I think, the test level of this PR is okay currently.

File: tests/ui/specs/Keypad/Keypad-specs.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			Page.spotlightRight();
+			expect(keypad.button(5).isFocused()).to.be.true();
+			Page.spotlightUp();
+			expect(keypad.button(2).isFocused()).to.be.true();
+			Page.spotlightLeft();
+			expect(keypad.button(1).isFocused()).to.be.true();
+		});
+	});
+
+	describe('5 way disabled', function () {
```
Comment (MikyungKim) [inline tests/ui/specs/Keypad/Keypad-specs.js:29]:
```suggestion
	describe('5-way disabled', function () {
```

File: tests/ui/specs/Keypad/Keypad-specs.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+const Page = require('./KeypadPage');
+
+describe('Keypad', function () {
+
+	beforeEach(function () {
+		Page.open();
+	});
+
+	describe('5 way', function () {
```
Comment (MikyungKim) [inline tests/ui/specs/Keypad/Keypad-specs.js:9]:
```suggestion
	describe('5-way', function () {
```

File: Keypad/tests/Keypad-specs.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import Keypad from '../Keypad';
+
+import css from '../Keypad.module.less';
+
+const click = (key, value) => key.find(`.${css.button}[aria-label="${value}"]`).simulate('click');
+
+describe('Keypad Specs', () => {
+	test('should add new digits on every digit click', () => {
```
Comment (MikyungKim) [inline Keypad/tests/Keypad-specs.js:10]: Could we add this test for the disabled one for not adding new digits?

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**